src/server/pfs/server/testing/server_test.go

```diff
@@ -7640,3 +7640,142 @@ func TestWalkFileInvalid(t *testing.T) {
 		return nil
 	}))
 }
+
+func gcOption(c *pachconfig.Configuration) {
+	c.PachdSpecificConfiguration.StorageGCPeriod = 5
+	c.PachdSpecificConfiguration.StorageChunkGCPeriod = 5
+}
+
+func TestForgetRPC(t *testing.T) {
+	ctx := pctx.TestContext(t)
+	dbcfg := dockertestenv.NewTestDBConfig(t)
+	env := realenv.NewRealEnv(ctx, t, dbcfg.PachConfigOption, gcOption)
+
+	require.NoError(t, env.PachClient.CreateRepo(pfs.DefaultProjectName, "in"))
+	require.NoError(t, env.PachClient.CreateRepo(pfs.DefaultProjectName, "out"))
+	require.NoError(t, env.PachClient.CreateBranch(pfs.DefaultProjectName, "out", "master", "", "", []*pfs.Branch{client.NewBranch(pfs.DefaultProjectName, "in", "master")}))
+
+	commitInfo, err := env.PachClient.InspectCommit(pfs.DefaultProjectName, "out", "master", "")
+	require.NoError(t, err)
+	commit := commitInfo.Commit
+
+	require.NoError(t, env.PachClient.PutFile(commit, "file", strings.NewReader("foo")))
+	require.NoError(t, finishCommit(env.PachClient, "out", "master", ""))
+	require.NoError(t, finishCommit(env.PachClient, "in", "master", ""))
+
+	var chunkCount int64
+	db := testutil.OpenDB(t, dbcfg.PGBouncer.DBOptions()...)
+	if err := dbutil.WithTx(ctx, db, func(cbCtx context.Context, tx *pachsql.Tx) error {
+		if err := db.QueryRow("select count(*) from storage.chunk_objects").Scan(&chunkCount); err != nil {
+			return errors.Wrap(err, "select storage chunks")
+		}
+		return nil
+	}); err != nil {
```

Comment:
[nit-pick]: You could make this more compact by writing it this way:
```go
err := dbutil.WithTX() ...
require.NoError(t, err)
```
and you can take that a step further by doing
```go
require.NoError(t, dbutil.WithTx(...)
```
I think if you can do the second one in this case. My reasoning is that, assuming someone breaks this test down the road, it should be fairly easy to figure out where the breakage is, even if its wrapped like that.